timemachine/cpp/src/context.cu

```diff
@@ -65,6 +65,16 @@ void Context::_verify_box(const double *box_buffer, cudaStream_t stream) {
             }
         }
     }
+
+    const double max_box_dim = *std::max_element(box_buffer, box_buffer + 3 * 3);
+    const auto [min_coord, max_coord] = std::minmax_element(coords_buffer, coords_buffer + N_ * 3);
+    // Look at the largest different in a single dimension, since coordinates are not imaged into the home box
+    // per say, rather into the nearest periodic box
```

Comment:
nits:
1) grammar: // per say -> per se
2) grammar:  // different -> difference
3) the comment says to look at the largest difference in a single dimension, but the actual operation looks over *all dimensions* since the offset is `N_*3`

Reply:
Addressed in https://github.com/proteneer/timemachine/pull/1401/commits/74585eccf9fb7955be8294ad049c367127b35180